kili/graphql_client.py

```diff
@@ -13,7 +13,7 @@
 from enum import Enum
 
 import websocket
-from six.moves import urllib
+from six.moves import urllib  # type:ignore
```

Comment:
you can remove six import, this is used for Python 2 compatibility with Python 3